### DISCUSSION TITLE
Show table from options value

### DISCUSSION THREAD

**Discussion by aloreaux** [Budibase/budibase#6853] [Help] [answered]
I am trying to have a drop down picker with certain values to send to a query and make it appear on screen. I have the drop down working I believe. I have it set to "execute query" but not sure if my bindings are working properly and also not sure how to then display the data in a table. Thanks

**Comment by melohagan** [comment, marked as answer]
Hey @aloreaux 

If you want to display data in a table then you'll need to use a datasource in a data provider. E.g. use a <a href="https://docs.budibase.com/docs/rest-bindings#data-provider-applications-provider">REST query</a> / <a href="https://docs.budibase.com/docs/custom-queries">custom query</a>

**Reply by aloreaux** [reply]
So I have the datasource for the table. What I am trying to accomplish is depending on the value of the dropdown menu it changes the query which would change the table values. The drop down is a bunch of names. If as certain name is selected the dropdown should 'execute query' and the new table should be displayed.

**Reply by melohagan** [reply]
If you use a binding in your Data Provider filter or a binding in your REST query (see docs) that points to the option picker field, then the data provider will automatically refresh when the value of that binding is changed.

**Reply by aloreaux** [reply]
Thanks! I got it working in a way. But once a selection is made it is stuck on that selection and I am unable to make another.

**Reply by aloreaux** [reply]
So I understand why it is doing this. Is it possible to have two data sources and keep the binding between the two?

**Reply by aloreaux** [reply]
The answer is yes. As long as the bindings are setup properly it can take the value from another query.

Thanks!
